src/extensionFunctions.ts

```diff
@@ -603,6 +603,15 @@ export namespace extensionFunctions {
 		}
 	}
 
+	async function openDidactOutputChannel() : Promise<void> {
+		if (!didactOutputChannel) {
+			didactOutputChannel = vscode.window.createOutputChannel(DIDACT_OUTPUT_CHANNEL);
+		}
+		if (didactOutputChannel) {
+			didactOutputChannel.show();
+		}
```

Comment:
can we simplify it with:

```
		if (!didactOutputChannel) {
			didactOutputChannel = vscode.window.createOutputChannel(DIDACT_OUTPUT_CHANNEL);
		}
		didactOutputChannel.show();
```

the `vscode.window.createOutputChannel `is always returning an outputchannel, isn't it?